stable/jenkins/templates/_helpers.tpl

```diff
@@ -164,6 +124,53 @@ unclassified:
     url: {{ template "jenkins.url" . }}
 {{- end -}}
 
+{{/*
+Returns kubernetes pod template configuration as code
+*/}}
+{{- define "jenkins.casc.podTemplate" -}}
+- name: "{{ .Values.agent.podName }}"
+  containers:
+  - name: "{{ .Values.agent.sideContainerName }}"
+    alwaysPullImage: {{ .Values.agent.alwaysPullImage }}
+    {{- if .Values.agent.args }}
+    args: "{{ .Values.agent.args }}"
+    {{- else }}
+    args: "^${computer.jnlpmac} ^${computer.name}"
+    {{- end }}
```

Comment:
`args: "{{ .Values.agent.args | replace "$" "^$" }}"` after rebase.

Reply:
Got it. Thanks for pointing it out.